rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java

```diff
@@ -944,6 +944,38 @@ public void should_document_get_projects_by_externalIds() throws Exception {
                         )));
     }
 
+    @Test
+    public void should_document_get_linked_projects() throws Exception {
+        String accessToken = TestHelper.getAccessToken(mockMvc, testUserId, testUserPassword);
+        mockMvc.perform(get("/api/projects/" + project.getId() + "/linkedProjects")
+                .header("Authorization", "Bearer " + accessToken)
+                .param("page", "0")
+                .param("page_entries", "5")
+                .param("sort", "name,desc")
+                .accept(MediaTypes.HAL_JSON))
+                .andExpect(status().isOk())
+                .andDo(this.documentationHandler.document(
+                        requestParameters(
+                                parameterWithName("page").description("Page of releases"),
```

Comment:
I hope it should be projects instead of releases

Reply:
made the changes.